tools/flaky-test-reporter/github_issue.go

```diff
@@ -121,6 +122,12 @@ func Setup(githubToken string) (*GithubIssue, error) {
 	return &GithubIssue{user: ghUser, client: ghc}, nil
 }
 
+// The Repo field of an github Issue could be empty, use URL is more reliable
```

Comment:
Nit: Should this method be renamed if its possible that repo could be empty?

Reply:
Even a valid issue can have empty Repo field, it's just a weird behavior of *github.Issue struct